drivers/bus/isapnp/hardware.c

```diff
@@ -740,48 +696,40 @@ ParseTags(
 
             case ISAPNP_TAG_STARTDEP:
             {
-                if (LogDev != 0 || TagLen > 1 ||
-                    NumberOfDepedentSet >= ISAPNP_MAX_ALTERNATIVES)
-                {
+                if (LogDev != 0)
                     goto SkipTag;
+
+                if (TagLen > 1)
+                {
+                    DPRINT1("Invalid tag %x\n", ISAPNP_TAG_STARTDEP);
+                    return STATUS_UNSUCCESSFUL;
                 }
 
                 if (DfState == dfNotStarted)
                 {
-                    LogDevice->Alternatives = ExAllocatePoolZero(PagedPool,
-                                                                 sizeof(ISAPNP_ALTERNATIVES),
-                                                                 TAG_ISAPNP);
-                    if (!LogDevice->Alternatives)
-                        return STATUS_INSUFFICIENT_RESOURCES;
-
                     DfState = dfStarted;
                 }
-                else if (DfState == dfStarted)
-                {
-                    ++NumberOfDepedentSet;
-                }
                 else
                 {
-                    goto SkipTag;
+                    if (DfState != dfStarted)
+                        goto SkipTag;
                 }
 
-                ++LogDevice->Alternatives->Count;
+                Resource->Type = ISAPNP_RESOURCE_TYPE_START_DEPEDENT;
+                Resource++;
 
                 if (TagLen != 1)
                 {
-                    LogDevice->Alternatives->Priority[NumberOfDepedentSet] = 1;
+                    Resource->Prioprity = 1;
```

Comment:
This member should be called `Priority`.

Reply:
Fixed.